### PR TITLE
feat: add min_instances parameter. 

### PR DESCRIPTION
**Pull request by savioteles**
This PR adds the min_instances parameter. The limit on the minimum number of fnction instances that may coexist at a given time.

**Reference**
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloudfunctions_function#min_instances

### CODE REVIEW

**Comment by savioteles** [comment]
@bharathkkb can you review? This PR adds the `min_instances` parameter.

File: main.tf [hunk 1]
```diff
@@ -85,6 +85,7 @@ module "main" {
   service_account_email              = var.function_service_account_email
   timeout_s                          = var.function_timeout_s
   max_instances                      = var.function_max_instances
+  min_instances                      = var.function_min_instances
```
Comment (bharathkkb) [inline main.tf:88]: We will need to add support for this in the [event-function module](https://github.com/terraform-google-modules/terraform-google-event-function) first

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR and sorry for the delay @savioteles

**Pull request closed without merging**